src/pages/lib/graphqlapi/subscribe-data/q/platform/[platform].mdx

```diff
@@ -106,7 +106,7 @@ deleteSub.unsubscribe();
 </Block>
 </BlockSwitcher>
 
-## Filter real-time subscriptions server-side
+## Set up server-side subscription filters
 
 Subscriptions take an optional `filter` argument to define service-side subscription filters:
```

Comment:
@renebrandel Should this be server-side or service-side? If they are interchangeable we should choose one (Heather in chat with Jim)